src/azure-cli-core/azure/cli/core/auth/landing_pages/success.html

```diff
@@ -2,11 +2,29 @@
 <html>
 <head>
     <meta charset="utf-8" />
-    <meta http-equiv="refresh" content="10;url=https://docs.microsoft.com/cli/azure/">
+    <meta http-equiv="refresh" content="120;url=https://docs.microsoft.com/cli/azure/">
     <title>Login successfully</title>
+    <style>
+        body {
+            font-family: 'Segoe UI', Tahoma, Geneva, Verdana, sans-serif; 
+        }
+
+        code {
+            font-family: 'Courier New', Courier, monospace;
+            display: inline-block;
+            font-weight: bold;
+            background-color: rgb(229, 229, 229);
```

Comment:
Use the same color as our doc:

```suggestion
            background-color: rgb(242, 242, 242);
```